changelogs/fragments/1491-gitlab-runner-owned-parameter.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - gitlab_runner - add ``owned`` param to allow non-admin use (https://github.com/ansible-collections/community.general/pull/1491)."
```

Comment:
```suggestion
  - gitlab_runner - add ``owned`` param to allow non-admin use (https://github.com/ansible-collections/community.general/pull/1491).
```

Reply:
You also need to add a `bugixes:` entry for the bug fix included in this new feature, namely fixing compatibilty with some python-gitlab versions.

Reply:
Also, please either write `parameter` out, or use `option`. Don't use `param`. That's not an English word.

Reply:
We should call it `option` according to the documentation